mesonbuild/ast/interpreter.py

```diff
@@ -378,15 +378,15 @@ def quick_resolve(n: BaseNode, loop_detect: T.Optional[T.List[str]] = None) -> T
             mkwargs = {} # type: T.Dict[str, TYPE_nvar]
             try:
                 if isinstance(src, str):
-                    result = StringHolder(src, T.cast(Interpreter, self)).method_call(node.name, margs, mkwargs)
+                    result = StringHolder(src, T.cast('Interpreter', self)).method_call(node.name, margs, mkwargs)
```

Comment:
I think you can save the hassle here and add `from __future__ import annotations`.

Reply:
The commit message for this individual change says:
> This is not possible to do with T.cast though, because it is a function argument instead of an annotation.

Reply:
Whoopssss